apps/ride-the-lightning/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  web:
+    image: shahanafarooqui/rtl:0.10.0@sha256:036d94c582911caa74cbe8a0f5646d98b3549010bdbdd2cebd92d3b08a00d82a
+    user: "1000:1000"
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 5m
+    ports:
+      - "$APP_RIDE_THE_LIGHTNING_PORT:$APP_RIDE_THE_LIGHTNING_PORT"
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+      - ${LND_DATA_DIR}:/lnd:ro
+      - ${BITCOIN_DATA_DIR}:/bitcoin:ro
+    environment:
+        # App config
+        PORT: $APP_RIDE_THE_LIGHTNING_PORT
+        RTL_CONFIG_PATH: "/data"
+        CHANNEL_BACKUP_PATH: "/data/backup"
+        LN_IMPLEMENTATION: "LND"
+
+        # Bitcoin Core
+        BITCOIND_CONFIG_PATH: "/bitcoin/bitcoin.conf"
```

Comment:
Also can we safely remove this?

Looks like it's only required to show the Bitcoin Core config in the settings page. Umbrel users don't really need to see this and it's better if apps that don't strictly need to read Bitcoin Core's data dir don't have access to it.

Reply:
Yeah, seems like it's only used here: 

https://github.com/Ride-The-Lightning/RTL/blob/0ec41f2de13aae7529cc3a13cb9a53aaaf0210e5/src/app/shared/components/settings/settings.component.ts#L61-L63